developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md

```diff
@@ -867,7 +867,47 @@ FAILURE proving 1 modules from package `hello_prover` in 0.067s
 ```
 In this case, see [Install the dependencies of Move Prover](install-aptos-cli#step-3-optional-install-the-dependencies-of-move-prover).
 
-### Debug and print stack trace
+### Profiling gas use
```

Comment:
Not a native speaker myself so I'm not 100% sure, but isn't "gas usage" more natural than "gas use"? I mean, you hear people talk about "electricity/gas/water usage" in real life.